docker_images/diffusers/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM tiangolo/uvicorn-gunicorn:python3.8
+LABEL maintainer="Nicolas Patry <nicolas@huggingface.co>"
+
+# Add any system dependency here
+# RUN apt-get update -y && apt-get install libXXX -y
+
+RUN pip install --no-cache-dir torch==1.12.0+cpu torchvision==0.13.0+cpu torchaudio==0.12.0 --extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
Why is this here rather than in `requirements.txt`?

Reply:
Docker cache hit. (Subsequent docker builds are much faster)